design/specs/foreign-key-constraints.md

```diff
@@ -176,6 +180,14 @@ The record preview allows users to identify records linked in other tables. It s
 - Multi-column unique constraints are potentially useful, but only if all columns are included.
 - Non-unique columns are never useful unless a unique column is included (that gives info for the user to choose between rows with duplicate entries in non-unique columns).
 
+### Linked Records for Multi-Column Foreign Key Constraints
+
+- The current UI does not take into account multi-column foreign key constraints and how those would be retrieved and selected.
```

Comment:
@ghislaineguerin Users might connect a database with multi-column foreign key constraints. How will we treat these? I assume we won't show any autocomplete behavior but it would be good to make that explicit.

Reply:
@kgodey can we say 'Unsupported table link' instead of showing the regular table link options on the column header menu